source/dem/particle_particle_nonlinear_force.cc

```diff
@@ -190,21 +206,9 @@ ParticleParticleHertzMindlinLimitOverlap<dim>::
                     particle_two_tangential_torque,
                     rolling_resistance_torque);
 
-                  // Getting particles' torque and force
-#if (DEAL_II_VERSION_MAJOR < 10 && DEAL_II_VERSION_MINOR < 4)
-                  types::particle_index particle_one_id =
-                    particle_one->get_id();
-                  types::particle_index particle_two_id =
-                    particle_two->get_id();
-#else
-                  types::particle_index particle_one_id =
-                    particle_one->get_local_index();
                   types::particle_index particle_two_id =
                     particle_two->get_local_index();
-#endif
-                  Tensor<1, 3> &particle_one_torque = torque[particle_one_id];
                   Tensor<1, 3> &particle_two_torque = torque[particle_two_id];
-                  Tensor<1, 3> &particle_one_force  = force[particle_one_id];
```

Comment:
Same as for the linear forces, is the directives at line 312 still relevant if you removed those here?